ui/tests/acceptance/secrets/backend/kv/secret-test.js

```diff
@@ -255,8 +255,8 @@ module('Acceptance | secrets/secret/create', function(hooks) {
     let paths = [
       '(',
       ')',
-      //'"',
-      "'",
+      '"',
+      //"'",
```

Comment:
why'd you skip this test?

Reply:
We had previously skipped both of them because they weren't supported in the web CLI - we do use those values for manual input into the regular KV ui below though. With this change we could do `"` because we now require that you quote the whole path. But we're quoting the path with `'` which doesn't work when the path also contains `'`.